src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
 // write code here
+
+const lis = document.querySelectorAll('li');
+
+for (const li of lis) {
+  const span = document.createElement('span');
+
+  span.addEventListener('click', (e) => {
+    const elem = e.currentTarget.nextSibling;
```

Comment:
The `nextSibling` property may return a text node (such as a newline or space) instead of an element node. Consider using `nextElementSibling` to ensure you get the next element node.